### DISCUSSION TITLE
Install the Sealed Commerce Scale Unit on my local VM

### DISCUSSION THREAD

**Discussion by ChristophNagel** [microsoft/Dynamics365Commerce.Solutions#340] [Q&A]
I need the id for my local DEV VM. Is there any environment Id for local deployed DFO? 

Otherwise i can not download the config file for the installation of the sealed CSU as descripted in  https://learn.microsoft.com/en-us/dynamics365/commerce/dev-itpro/install-csu-dev-env#create-a-new-channel-database-record

**Comment by AzibZafar** [comment]
Please check out this blog, which also includes a video. Hopefully, it will help resolve your issue.

https://www.linkedin.com/pulse/install-commerce-scale-unit-development-environment-valery-plotnik-mgjcc/?trackingId=QYcmHbvQQ%2FSr%2Fe26sFejpQ%3D%3D&trk=article-ssr-frontend-pulse_little-text-block
